e2e/nomostest/new.go

```diff
@@ -78,29 +78,40 @@ func newOptStruct(testName, tmpDir string, t nomostesting.NTB, ntOptions ...ntop
 		opt(optsStruct)
 	}
 
-	e2eTest := !optsStruct.LoadTest && !optsStruct.StressTest
-	if !*e2e.E2E && e2eTest {
+	if !*e2e.E2E {
```

Comment:
nit: I think this could be moved to input validation in `main_test.go`

Reply:
Removed. `main_test.go` already has a check for it.